apps/frontend/src/ViewFormsList2/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import {Box, Center, Heading, Select, Table, Tbody, Td, Th, Thead, Tr} from '@chakra-ui/react';
+import testData from "../testData.json"
+
+export default function ViewFormsList2 () {
+  const [sortBy, setSortBy] = useState('name');
+  let childData = testData;
+
+  if (sortBy === 'name') {
+    childData = testData.sort((a, b) => a.childsName.localeCompare(b.childsName));
```

Comment:
We should probably create a type here in the shape of a form entry to make TypeScript happy.